scripts/install_bitcoind.sh

```diff
@@ -2,9 +2,9 @@
 
 set -ev
 
-BITCOIND_VERSION=${BITCOIN_VERSION:-25.0}
+BITCOIND_VERSION=${BITCOIN_VERSION:-25}
 
 docker pull lightninglabs/bitcoin-core:$BITCOIND_VERSION
 CONTAINER_ID=$(docker create lightninglabs/bitcoin-core:$BITCOIND_VERSION)
-sudo docker cp $CONTAINER_ID:/opt/bitcoin-$BITCOIND_VERSION/bin/bitcoind /usr/local/bin/bitcoind
+sudo docker cp $CONTAINER_ID:/opt/bitcoin-$BITCOIND_VERSION.0/bin/bitcoind /usr/local/bin/bitcoind
```

Comment:
I guess this assumes we gonna always build with major versions?

Reply:
Yeah, which I think is okay for our CI. We'll notice quite quickly if this becomes an issue.